docs/guide/yaml/test/usage-examples.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Example Blueprint Tests
+title_in_menu: Example Tests
+layout: website-normal
+---
+
+{% include fields.md %}
+
+## Introduction
+This section provides example tests carried out on the following `simple-tomcat-app` web application catalog item:
+
+{% highlight yaml %}
+{% readj example_yaml/simple-tomcat-app.yaml %}
```

Comment:
Do we want to show all these examples as catalog items, or just as YAML for deploying an app directly? The latter would be shorter examples.